projects.yaml

```diff
@@ -61,133 +64,238 @@ projects:
         - *module_bl
         - *module_hic_kl26z
         - records/board/kl26z_bl.yaml
+    kl26z_if:
+        - *module_if
+        - *module_hic_kl26z
+        - records/family/nxp_kinetis_kseries.yaml
+        - records/family/nxp_kinetis_lseries.yaml
+        - records/family/nxp_mimxrt.yaml
+        - records/family/nordic_nrf51.yaml
+        - records/family/nordic_nrf52.yaml
+        - records/family/realtek_rtl8195am.yaml
+        - records/family/ti_family.yaml
+        - records/family/wiznet_family.yaml
```

Comment:
Why don't you create a yaml module for all families, so you don't have to duplicate the family yaml list for each _if project?